thesheriff/infrastructure/repository/mysql_raid_repository.py

```diff
@@ -103,10 +103,11 @@ def __split_outlaw_ids(self, outlaws: str) -> List[int]:
         return [int(id) for id in ids]
 
     def __create_outlaw_from_ids(self, members: list) -> List[Outlaw]:
-        # results = list()
-        #  for member in members:
-        #     results.append(
-        #         OutlawFactory.create(
-        #         )
-        #     )
-        pass
+        # FIXME: find outlaws by id
+        results = list()
```

Comment:
May we get all the outlaws and filter it manually by the id (i.e member.id through OutlawRepo.all()?

Reply:
My doubt was where to put the OutlawRepo, would that be here (repo as collaborator of another repo?) or on  the use case itself?

Reply:
I guess better at UseCase layer, we are sure we can have more than one repo as a collaborator ... but I've never seen before a repo as a collaborator for another repo